lib/kintsugi/apply_change_to_project.rb

```diff
@@ -626,6 +665,11 @@ def add_build_file(build_phase, change, change_path)
         return
       end
 
+      duplicate_build_file_exists = !build_phase.files.find do |build_file|
+        build_file.file_ref.path == change["fileRef"]["path"]
+      end.nil?
+      return if !Settings.allow_duplicates && duplicate_build_file_exists
+
```

Comment:
it's a bit hard to read when the `!` is in the beginning and the `.nil?` is in the end.
consider
```suggestion
      duplicate_build_file = build_phase.files.find do |build_file|
        build_file.file_ref.path == change["fileRef"]["path"]
      end
      return if !Settings.allow_duplicates && !duplicate_build_file.nil?
```

same in other places

Reply:
maybe `existing_build_file` instead of `duplicate_build_file`